rdkit/Chem/Scaffolds/MurckoScaffold.py

```diff
@@ -75,6 +75,7 @@ def GetScaffoldForMol(mol):
     res.UpdatePropertyCache()
     Chem.GetSymmSSSR(res)
   else:
+    # This code cannot be reached
     res = _pyGetScaffoldForMol(mol)
```

Comment:
The original Python implementation was at some point replaced by a C++ implementation. @greglandrum, what do you think about removing the old code?

Reply:
In general we've left the python implementations in place. I suppose there's not really an argument against removing them at some point. That's a different PR though. :-)